libqtile/layout/xmonad.py

```diff
@@ -299,6 +305,19 @@ def configure(self, client, screen):
         self._configure_specific(client, screen, px, cidx)
         client.unhide()
 
+    def _fullscreen_no_border(self, *args):
+        client = self.clients[self.focused]
+        if client.window.get_net_wm_state() == ['fullscreen']:
```

Comment:
I'm not sure I understand why this is xmonad specific; shouldn't we remove the border for all fullscreened windows?

Reply:
I dont think it is xmonad specific, but by default xmonad does have border even on fullscreen windows, the wierd part is, right now its a bit bugged, when you go fullscreen it doesnt show up 'all' borders, let me bring a screenshot so you can see what im saying.
![2020-07-14_07-50](https://user-images.githubusercontent.com/8454169/87417607-d4950380-c5a6-11ea-97c0-68a70e2b6fe7.png)

but yeah i hate those borders even on default xmonad

Reply:
I get the fullscreen borders sometimes (not always) in the Columns layout, though this only started happening in the last couple months

Reply:
I wonder if the "not always" bug is related to the user-specified position fixes.

But anyway, this all seems like a bug to me in window.place(); if a window is fullscreen, we should just set the border width to 0 there, right? No need for layout specific behavior.